phenotrex/ml/cccv.py

```diff
@@ -107,7 +116,7 @@ def _completeness_cv(self, param, **kwargs) -> Dict[float, Dict[float, float]]:
         subprocesses using a ProcessPoolExecutor from concurrent.futures
 
         :param param: List [test_records, X_train, y_train, comple_steps, conta_steps, starting_message]
-            workaround to get multiple parameters into this function. (using processor.map) #TODO find nicer solution?
+            workaround to get multiple parameters into this function. (using processor.map)
```

Comment:
Have a look at `itertools.starmap` and `multiprocessing.Pool.starmap` which should do that, iirc.

Reply:
Thanks, good to know! It appears that `starmap` is currently not implemented in the `concurrent.futures` interface to `multiprocessing` so this would require some larger code changes for a little more readability. I'll keep it in mind though.

Reply:
I've never used concurrent. You'll have to explain the advantages over multiprocessing at some point :)